### DISCUSSION TITLE
Exclude "today" from the last X days

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1482] [Anything Else]
> I am always surprised that the time period options beginning with "last" (like "last 7 days") include "today"'s incomplete data. Even in "month to date" I would prefer to only see complete days, but at least there you can argue for semantical correctness, whereas I would never consider "today" part of the "last 7" days.  "Last", to me, implies "previous".
> 
> And, relevant to that, seeing that "today"'s incomplete data is "down" X% isn't really helpful or accurate?  The data is just incomplete, an accurate comparison cannot be made...unless you are comparing "today" hour by hour with the previous period?

(Feedback via email)

**Comment by metmarkosaric** [comment]
we also have the thread for fixing the today comparison to compare to the same amount of time yesterday rather than the full day yesterday #344
